controller/engine_controller.go

```diff
@@ -973,11 +989,23 @@ func (m *EngineMonitor) refresh(engine *longhorn.Engine) error {
 			m.logger.Debug("Cannot restore the backup for engine since it is still in the backoff window")
 			return nil
 		}
+
+		if isReachedLimit, err := m.isReachedConcurrentVolumeBackupRestoreLimit(); err != nil {
+			m.logger.WithError(err).Warn("Failed to check concurrent volume backup restore limit")
+			return nil
+
+		} else if isReachedLimit {
+			m.logger.Debugf("Reached the concurrent volume backup restore limit of %v, retry later", types.SettingNameConcurrentBackupRestorePerNodeLimit)
```

Comment:
Should it be warning level message?

Reply:
I see. Just thought if we can easily know why the backup does not take place from log.